src/myframe.h

```diff
@@ -1070,6 +1070,7 @@ struct MyFrame : wxFrame {
                 TSCanvas *p = (TSCanvas *)nb->GetPage(i);
                 p->doc->dpichanged = true;
                 }
+        Update();

```

Comment:
Fix indentation of this function?
Why does each doc have its own `dpichanged` ? Can it different for different docs? Otherwise move to System or Frame?

Reply:
Thanks for your feedback. Yes, `dpichanged` signals that the layout and the children must be reset and this is per document. Othewise you may need to be make this a global state and thus check when the last document has been successfully painted? This adds complexity somewhere elese I think.

Reply:
ok, lets leave it then!